### ISSUE TITLE
wrong pixel size in particle extraction protocol

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-em-relion#31]
```
01484:    Joining metadata of all particles from 1425 micrographs in one STAR file...
01485:    Written out STAR file with 413146 particles in micrographs_00001-01425_particles.star
01486:    The new pixel size of the extracted particles are 2.27079e-316 Angstrom/pixel.
01487:    Done preprocessing!
01488:   FINISHED: extractMicrographListStep, step 1
```

Problem: rlnDetectorSize is missing in input star file, while rlnMagnification exists

**Comment by azazellochg** [comment]
@delarosatrevin I'm thinking just to add --set_angpix arg to cmd in this protocol. The pixel size is in cmd for most other commands in relion, except this one

**Comment by delarosatrevin** [comment]
Ok, great! Would you mind to add a small test for it...that fails before this fix and not now?

**Comment by azazellochg** [comment]
nothing fails, in fact, it's just relion prints a wrong number in the output log

**Comment by azazellochg** [comment]
fixed in 43b0b29
